jetpack.php

```diff
@@ -75,6 +75,25 @@ function jetpack_should_use_minified_assets() {
 	}
 	return true;
 }
+
+/**
+ * Outputs for an admin notice about running Jetpack on outdated WordPress.
+ *
+ * @since 7.2.0
+ */
+function jetpack_admin_unsupported_wp_notice() { ?>
+	<div class="notice notice-error is-dismissible">
+		<p><?php esc_html_e( 'Jetpack requires a more recent version of WordPress and has been paused. Please update WordPress to continue enjoying Jetpack.', 'jetpack' ); ?></p>
```

Comment:
Would it make sense to provide the user with a link to support to help them achive updating of WordPress. 
also does do we show this to everyone. 
Even non admins?

Reply:
As it stands, it shows to everyone. I'm okay not changing behavior though because it would only been shown on sites that upgraded to Jetpack in some atypical way and with Jetpack not working, it would be pretty apparent to anyone who used any feature.